cloudbuild.yaml

```diff
@@ -396,15 +407,21 @@ steps:
             # "Complete" is successful job run.
             # Version 1.31 has "SuccessCriteriaMet" as the first completion status returned, or "FailureTarget" in case of failure.
             if [ "$output" == "Complete" ] || [ "$output" == "SuccessCriteriaMet" ] ; then
+              echo "Job completed successfully on cluster associated with log: $outputLog"
               continue
             else
+              echo "Unexpected job status: '${output}' in log ${outputLog}"
+              echo "Collecting diagnostics from failed job..."
+              kubectl describe job upgrade-test-runner | awk '/Events:/,/^$/' | tail -n +2
```

Comment:
There might be an issue here, it will always get describe the job from the latest kube context, we might need to keep track of the cluster name + region to be able to run `gcloud container clusters get-credentials "$testCluster" --region="$testClusterLocation" --project="$PROJECT_ID"`  before calling it, otherwise it would only use the latest context set, what do you think ?